src/components/questionStructure/Task/variants/handwritingTasks/RawHandwritingTask.tsx

```diff
@@ -0,0 +1,50 @@
+import { Pencil2Icon } from '@radix-ui/react-icons'
+import { Button, Dialog, Flex } from '@radix-ui/themes'
+import { isEmpty } from 'lodash'
+import { FC } from 'react'
+
+import { TaskType } from '../../constants'
+import { TaskBaseProps } from '../../types'
+import { ViewOnlyCanvas } from './components/ViewOnlyCanvas'
+import RawHandwritingEditor from './editors/RawHandwritingEditor'
+import { RawHandwritingAnswer } from './types'
+
+export interface RawHandwritingProps extends TaskBaseProps<RawHandwritingAnswer> {
+  type: TaskType.RAW_HANDWRITING
+}
+
+export const RawHandwritingTask: FC<RawHandwritingProps> = ({
+  answer,
+  onAnswerUpdate,
+  disabled = false,
+}) => (
+  <Dialog.Root>
+    {!disabled && (
+      <Flex gap="3" align="center">
+        <Dialog.Trigger>
+          <Button disabled={disabled} size="4">
+            {answer?.raw?.elements.length ? 'Edit answer' : 'Enter answer'}{' '}
```

Comment:
We actually tried - unfortunately it's more complicated than you think.

You have to ensure the answer & onAnswerUpdate props match the underlying Editor you pass in, which means generics, and we found the generic non-trivial to do (since you're basically relying on TS infering the types correctly).

So we gave up and decided to do it another day as it looked to be quite complex.

Also they're not quite identical - one has a latex preview, the other does not and this preview exists outside of the Editor component!